### PR TITLE
Semaphore heroku setup

### PR DESCRIPTION
**Pull request by DenisZackharov**
Add Semaphore config for auto deploy on heroku

### CODE REVIEW

File: .semaphore/heroku.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    os_image: ubuntu1804
+blocks:
+  - name: Deploy
+    task:
+      secrets:
+        - name: preinternship
+      env_vars:
+        - name: HEROKU_REMOTE
+          value: 'https://git.heroku.com/fierce-plains-93658.git'
+        - name: HEROKU_APP_NAME
```
Comment (sergeyantonov1) [inline .semaphore/heroku.yml:15]: I'm not sure that HEROKU_APP_NAME &  HEROKU_API_KEY are requred keys. Could you check, please?

File: .semaphore/semaphore.yml [hunk 2]
```diff
@@ -1,26 +1,67 @@
...
+  queued:
+    when: 'true'
+fail_fast:
+  stop:
+    when: 'true'
+global_job_config:
+  env_vars:
+    - name: RAILS_ENV
+      value: test
+    - name: DATABASE_URL
```
Comment (sergeyantonov1) [inline .semaphore/semaphore.yml]: seems like, DATABASE_URL is redundant. Could you check please?

File: .semaphore/semaphore.yml [hunk 3]
```diff
@@ -1,26 +1,67 @@
...
-      env_vars:
-        - name: DATABASE_URL
-          value: "postgresql://postgres:@0.0.0.0:5432/myapp_test"
-        - name: RAILS_ENV
-          value: test
       jobs:
-      - name: Test
+        - name: Quality
+          commands:
+            - 'bundle install ${BUNDLE_ARGS} --local'
```
Comment (sergeyantonov1) [inline .semaphore/semaphore.yml]: please, see documentation about caching https://docs.semaphoreci.com/essentials/caching-dependencies-and-directories/

File: .semaphore/heroku.yml (hunk 1 above)
Comment (DenisZackharov) [reply .semaphore/heroku.yml:15]: HEROKU_API_KEY need for complete rails db:migrate, HEROKU_APP_NAME deleted

**Pull request merged**